include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -326,6 +326,56 @@ def TTIR_ReluOp : TTIR_ElementwiseUnaryOp<"relu"> {
     }];
 }
 
+def TTIR_RoundOp : TTIR_DPSOp<"round"> {
```

Comment:
What's the rationale behind adding these two as separate ops in TTIR dialect? They have a similar behaviour that can be captured with one bool or enum that can later be interpreted during lowering.